src/commands/src/shared/next-gen-stage.ts

```diff
@@ -0,0 +1,46 @@
+import { executeCliCommand, getCommitHash, getPatchId } from '../helpers';
+
+interface StageParams {
```

Comment:
nit: would we want to just extend off of `NextGenParseParams` interface, since those types are shared with `StageParams`

Reply:
That's an option for sure, but I think it might be a bit more readable to list them all out. I also was thinking these interfaces to be isolated specifically for each function so they wouldn't need to reference another interface for another command.
 
 That's me being nitpicky though, haha! What are your thoughts?

Reply:
I think that is fair. and well thought out.